Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -425,24 +441,33 @@ int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int coun
                           primitive < OpcodeDecoder::Primitive::GX_DRAW_LINES);
 
     const int stride = loader->m_native_vtx_decl.stride;
-    DataReader dst = g_vertex_manager->PrepareForAdditionalData(primitive, count, stride,
-                                                                cullall || can_cpu_cull);
+    do
+    {
+      const int max_vertices = 16380;  // Max is 16383, but 16380 is divisible by both 4 and 3
+      int run = CanSplit(primitive) && count > max_vertices ? max_vertices : count;
+      count -= run;
+      DataReader dst = g_vertex_manager->PrepareForAdditionalData(primitive, run, stride,
+                                                                  cullall || can_cpu_cull);
 
-    count = loader->RunVertices(src, dst.GetPointer(), count);
+      run = loader->RunVertices(src, dst.GetPointer(), run);
```

Comment:
```suggestion
      const int vertices_processed = loader->RunVertices(src, dst.GetPointer(), run);
```

(is this value the vertices?  We update the stat `num_prims` below but the function `RunVertices()` looks like it operates in terms of vertices...)

Reply:
My understanding is that it's possible for some vertices to get skipped when loading, so this is the number of vertices that were written to `dst` (while the input is the number of vertices in `src`).  But I'm not super familiar with gamecube draw calls so it might be slightly different.
Maybe we should call this `num_loaded`?

Reply:
`num_loaded` would be fine or `vertices_loaded`?